docs/web-chat/sample-integration.mdx

```diff
@@ -0,0 +1,29 @@
+---
+sidebar_position: 10
+title: Sample Integrations
+description: "Sample integrations in some popular frontend frameworks."
+---
+
+import { Intro } from "@site/src/components/forDocs";
+
+<Intro>
+
+Here are some sample integrations in some popular frontend frameworks to help you adhere to best practices while integrating Web Chat into your website or web app.
+
+</Intro>
+
+## React
+
+An implementation guide to integrate Web Chat using react with source code is available [here](https://github.com/helpshift/helpshift-webchat-examples).
```

Comment:
@dv-Gogna @hs-umang Here the link is pointing to the home directory of the repository, instead, it should point to the react directory, right? Similarly for others.

Reply:
yes, we will change it once we release all the sample apps
Currently we don't have their links

Reply:
Ack